operations/locustfile.py

```diff
@@ -27,8 +37,11 @@ def get_health(self):
 
     @task(5)  # this task will get called 5x more than the other
     def post_v1_etor_demographics(self):
-        self.client.post("/v1/etor/demographics", data=request_body)
+        self.client.post("/v1/etor/demographics", data=demographics_request_body)
 
+    @task(5)  # this task will get called 5x more than the other
```

Comment:
Not sure if we need to call this test 5x. Realistically, I'm guessing it would be called a lot less than the demographics endpoint?

Reply:
I updated this to 1 (removed the 5) based on what @basiliskus and I discussed.